ServerCore/Pages/Teams/PlayProto.cshtml

```diff
@@ -0,0 +1,31 @@
+﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/PlayProto"
+@model ServerCore.Pages.Teams.PlayProtoModel
+
+@{
+    @using Helpers;
+
+    ViewData["Title"] = "Puzzles";
+    ViewData["AdminRoute"] = "/Puzzles/Index";
+    ViewData["AuthorRoute"] = "/Puzzles/Index";
+}
+
+<h2>Puzzles</h2>
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                <div>Puzzle</div>
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        @foreach (var puz in Model.PuzzleViews)
+        {
+            <tr>
+                <td>
+                    <a asp-page="/Submissions/SubmitProto" asp-route-puzzleId="@puz.ID">@RawHtmlHelper.Display(puz.Name, Model.Event.ID, Html)</a>
```

Comment:
Is this the right `asp-page` value? Looks like you deleted `Submissions` in the first line of this review.

Reply:
It goes to the right place in my local testing.  From what I can tell, that @page path at the top of the file is what the URL _shows_, but the path to the file based on our directory structure is what this link uses to locate what file to actually go to.  They don't necessarily have to match.  In fact, when I had this link use the @page path, it didn't work!  You can also see that this is exactly what our existing pages do:  the Play page goes to the Index page according to the link, but the actual Index page itself never shows the word Index in its @page path, which is why I modelled this POC off of that.

Reply:
Oops, hope that guy I accidentally pinged with the @ ignores this!